### PR TITLE
Added grab handles for move and resize

### PR DESCRIPTION
**Pull request by bart-de-jonge**
See title

### CODE REVIEW

File: src/main/java/gui/TimetableBlock.java [hunk 1]
```diff
@@ -111,6 +112,33 @@
...
+                        break;
+                    case Resize_Bottom:
+                    case Resize_Top:
+                        newCursor = Cursor.N_RESIZE;
+                        break;
+                    case Resize_Left:
+                    case Resize_Right:
+                        newCursor = Cursor.E_RESIZE;
+                        break;
+
```
Comment (Markvanderuit) [inline src/main/java/gui/TimetableBlock.java]: random space here :)

File: src/main/java/gui/TimetableBlock.java [hunk 2]
```diff
@@ -111,6 +112,33 @@
...
+                }
+
+                if (currentCursor != newCursor) {
```
Comment (Markvanderuit) [inline src/main/java/gui/TimetableBlock.java]: just use getCursor here?

**Comment by Markvanderuit** [comment]
Looks good. Second member can merge.

**Comment by geenen124** [comment]
Looks good!

**Pull request merged**